### DISCUSSION TITLE
categorisation of mtt and sng

### DISCUSSION THREAD

**Discussion by jejellyroll-fr** [jejellyroll-fr/fpdb-3#57] [development]
hello @brunetton @sf-87 

I need your opinion because I'm working on the parser for room party FR (which could serve as a basis for improvement for the others).
As you play more of these formats (sng and mtt), you should have an idea on the subject:

Today, I don't see any real categorisation for these formats, but I suppose there is a need to filter the results and each room has its own way of naming its mtt or sng:

so I was thinking of making a config file in yaml format for each room (not hard-coded like the test I did in MergeStructures.py).

This will probably also involve modifying the database, particularly to integrate formats such as mystery mtt or lottery sbg.

I don't know if you've gone into these parts of the code, but what do you think?

**Comment by sf-87** [comment]
Hi @jejellyroll-fr 
speaking of Pokerstars how fpdb actually works is all I need: sng are divided by buyin (and eventually maxseat in case of mtt sng), while mtt are again divided by buyin, maxseat and others things like pko, reentry and stuff.
When I open the tourney stats tab I can see how I'm doing on every single category of them (ie: 1€ 6-max sng, 3€ 27-max mtt sng, 2€ 6-max mtt with reentry, 2€ 8-max pko mtt with reentry).

I'm not sure how other rooms work and if they save tournament name on TS (PS did NOT do it)
